stubs/3.2/queue.py

```diff
@@ -5,6 +5,7 @@
 from typing import Any
 
 class Queue():
+    @staticmethod
```

Comment:
I should have looked more careful in fixing the type error. The `self` parameter is missing here.